htdocs/admin/stock.php

```diff
@@ -52,14 +52,17 @@
 	// Mode of stock decrease
 	if ($action == 'STOCK_CALCULATE_ON_BILL'
 	|| $action == 'STOCK_CALCULATE_ON_VALIDATE_ORDER'
-	|| $action == 'STOCK_CALCULATE_ON_SHIPMENT')
+	|| $action == 'STOCK_CALCULATE_ON_SHIPMENT'
+	|| $action == 'STOCK_CALCULATE_ON_SHIPMENT_CLASSIFY_BILLED')
```

Comment:
Hi @eldy i named it like that because the action was added into set_billed method of the Expedition class. I'll rename it